cli/tests/unit_node/process_test.ts

```diff
@@ -365,6 +365,8 @@ Deno.test({
   name: "process.stdin readable with a TTY",
   // TODO(PolarETech): Run this test even in non tty environment
   ignore: !Deno.isatty(Deno.stdin.rid),
+  // stdin resource is present before the test starts.
+  sanitizeResources: false,
```

Comment:
Why is it needed?

Reply:
The comment above says it.

stdin resource is created before the test starts and closed inside the test - triggering the sanitizer